Wikipedia/Code/ErrorWorkaround.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ErrorWorkaround.swift
```

Comment:
Is there a better name for this, or some comments? I'm not sure what is being worked around.

Also It looks like you are extending `WMFImageController` - is this where that extension should be? We haven't really discussed swift style - so idk.